src/autocomplete/Components.tsx

```diff
@@ -70,7 +71,9 @@ export const PillCompletion = forwardRef<IPillCompletionProps, any>((props, ref)
             ref={ref}
         >
             {children}
-            <span className="mx_Autocomplete_Completion_title">{title}</span>
+            <span className="mx_Autocomplete_Completion_title">
+                {isEmote ? <img className="mx_Emote" src={title} alt="" /> : title}
```

Comment:
using `title` as `src` is confusing - `titleOrSrc` would equally be confusing. Maybe it should accept a `titleComponent` which takes precedence over `title`